client/checkout/classic/upe.js

```diff
@@ -38,6 +38,11 @@ jQuery( function ( $ ) {
 			forceNetworkSavedCards: getConfig( 'forceNetworkSavedCards' ),
 			locale: getConfig( 'locale' ),
 			isUPEEnabled,
+			apiVersion:
```

Comment:
It's more reliable, but we could bring a little bit more consistency here:

```javascript
const isStripeLinkEnabled = isUPEEnabled && paymentMethodsConfig.link !== undefined && paymentMethodsConfig.card;
```

And pass it into the constructor as this happens with `isUPEEnabled`.
Correspondingly the version injection is moving to `createStripe` method.